pages/common/grpcurl.md

```diff
@@ -0,0 +1,20 @@
+# grpcurl
+
+> Like cURL, but for gRPC: CLI tool for interacting with gRPC servers.
+> More information: <https://github.com/fullstorydev/grpcurl>.
+
+- Send an empty request:
+
+`grpcurl grpc.server.com:443 my.custom.server.Service/Method`
```

Comment:
Looks like some things here can be changed by the user. Let's use the `{{token_syntax}}` here.
```suggestion
`grpcurl {{grpc.server.com:443}} {{my.custom.server.Service/Method}}`
```